docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -30,22 +30,32 @@ Array.sum array
 ```
 
 #### Parameters
-*array*
-Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
-
+*array* Type: **^T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
 
 The input array.
 
 
+## Return Value
+
+The resulting sum.
+
 
-**The resulting sum.**
 ## Remarks
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception(v=vs.110).aspx), as the function uses the [checked (+) operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp]) for the additions.
```

Comment:
@Vandroiy The converter that generated the docs seems to have put the : inside of the bold - there are one or two files that have been hand-written/edited, which may have it the other way.  However, 99%+ have the colon in bold.

I agree that your change is probably "correct", but was just pointing out that it doesn't fit the styles in these docs.  @dend Do you have a preference here? I could probably fix the repo to follow Chicago Manual of Style globally using my script pretty easily....


@Vandroiy Re: hash instead of repo path - that's one that I'm not sure about - I don't know how to find/get those hashes, but the other style (as I pasted) works and is easy, so that's what I've been using when I tweak things.  The hashes were, again, from the original dump to the open source docs.